### ISSUE TITLE
Server Side Rendering won't work with default styles

### ISSUE THREAD

**Issue by texttechne** [reactjs/react-tabs#45]
Server side rendering won't work. I get the same error as described in rackt/react-tabs/issues/39. 
```
Tabs.js:81 [object Object].componentWillMount
[app]/[react-tabs]/lib/components/Tabs.js:81:37
```
Are react-tabs supposed to work on the server side? If not, any plans on supporting them?

**Comment by texttechne** [comment]
Disabling default styles solves the issue. 
```
ReactTabs.Tabs.setUseDefaultStyles(false);
```
So the way https://github.com/ryanflorence/js-stylesheet is used within Tabs component seems to be the fault in this case: https://github.com/rackt/react-tabs/blob/master/lib/components/Tabs.js#L60-L64

**Comment by zebapy** [comment]
Why isn't this just a prop?

**Comment by danez** [comment]
Because most probably you want to disable the styles globally for all instances of the tabs on your site.

This issue was fixed btw in 0.5
